examples/90_Kubernetes/README.md

```diff
@@ -1,19 +1,32 @@
 # Kubernetes
 
+<<<<<<< HEAD
```

Comment:
It seems that there was a merge conflict here and it wasn't cleanly resooved.

Reply:
Indeed.  I'm search for the new documentation I spent all morning writing!!